.travis.yml

```diff
@@ -11,8 +11,9 @@ android:
     - extra-google-m2repository
     - addon-google_apis-google-16
 before_script:
-  - cd actionlist && npm install
+  - cd actionlist && rm -rf node_modules && npm install
```

Comment:
@burnumd is it not ok to just delete the node modules folder? Our changes had no effect in the build failing.